app-i18n/rime-data-array/rime-data-array-0.0.0.20221120.ebuild

```
@@ -0,0 +1,43 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+MY_PN="rime-array"
+MY_COMMIT="b37aad383ff6e71e457aa6d1d47d2040af8649b9"
+
+DESCRIPTION="array input schema and dictionary for RIME"
+HOMEPAGE="https://rime.im/ https://github.com/rime/rime-array"
+SRC_URI="https://github.com/rime/${MY_PN}/archive/${MY_COMMIT}.tar.gz -> ${MY_PN}-${PV}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+
+DEPEND="!app-i18n/rime-data:0"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_PN}-${MY_COMMIT}"
+
+RIME_DATA_DIR="/usr/share/rime-data"
+DOCS=(
+	AUTHORS
+	LICENSE
+	README.md
+)
+
+src_install() {
+	insinto "$RIME_DATA_DIR"
```

As per https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101
Please change other ebuilds too.
```suggestion
	insinto "${RIME_DATA_DIR}"
```

---

In fact, there is no difference between them in the case of the use of a single variable, does the QA reports some error?

---

Quotes from PG0101
`Rationale: the recommended constructs are less error-prone. Consistency avoids unnecessary changes when other 
developers edit the ebuild.`

---

Okay